rockcraft.yaml

```diff
@@ -66,3 +66,25 @@ parts:
             dpkg-query -W -f "${dpkg_ops}\n" \
             --root=${CRAFT_PROJECT_DIR}/../parts/postgresql-snap/layer/ \
             >> $CRAFT_PRIME/usr/share/rocks/dpkg.query
+            ln -f $CRAFT_PRIME/usr/share/postgresql/14/extension/\
+            postgis-3.control \
+            $CRAFT_PRIME/usr/share/postgresql/14/extension/postgis.control
+            ln -f $CRAFT_PRIME/usr/share/postgresql/14/extension/\
+            address_standardizer-3.control $CRAFT_PRIME/usr/share/postgresql/\
+            14/extension/address_standardizer.control
+            ln -f $CRAFT_PRIME/usr/share/postgresql/14/extension/\
+            address_standardizer_data_us-3.control $CRAFT_PRIME/usr/share/\
+            postgresql/14/extension/address_standardizer_data_us.control
+            ln -f $CRAFT_PRIME/usr/share/postgresql/14/extension/\
+            postgis_raster-3.control $CRAFT_PRIME/usr/share/postgresql/14/\
+            extension/postgis_raster.control
+            ln -f $CRAFT_PRIME/usr/share/postgresql/14/extension/\
+            postgis_tiger_geocoder-3.control $CRAFT_PRIME/usr/share/\
+            postgresql/14/extension/postgis_tiger_geocoder.control
+            ln -f $CRAFT_PRIME/usr/share/postgresql/14/extension/\
+            postgis_topology-3.control $CRAFT_PRIME/usr/share/\
+            postgresql/14/extension/postgis_topology.control
+            export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CRAFT_PRIME/usr/lib/ \
+            $CRAFT_ARCH_TRIPLET/blas:$CRAFT_PRIME/usr/lib/ \
+            $CRAFT_ARCH_TRIPLET/lapack
+            export TCL_LIBRARY=$CRAFT_PRIME/usr/share/tcltk/tcl8.6
```

Comment:
What about to increase readability a bit:
```
...
>> $CRAFT_PRIME/usr/share/rocks/dpkg.query
# PostgreSQL extensions
PG14_EXT="${CRAFT_PRIME}/usr/share/postgresql/14/extension"
ln -f \
  "${PG14_EXT}/postgis-3.control" \
  "${PG14_EXT}/postgis.control"
ln -f \
  "${PG14_EXT}/address_standardizer-3.control" \ 
  "${PG14_EXT}/address_standardizer.control"
...
```


OR

```
...
>> $CRAFT_PRIME/usr/share/rocks/dpkg.query
# PostgreSQL extensions
pushd "${CRAFT_PRIME}/usr/share/postgresql/14/extension"
ln -f postgis-3.control postgis.control
ln -f address_standardizer-3.control address_standardizer.control
...
popd
```

P.S. please add separation comment to the top of the section, otherwise it is mixing with section "# Generate dpkg.query".

As usually please add PR description and ticket to the PR title. Tnx!